doc/getting_started/local_installation.md

```diff
@@ -89,8 +89,8 @@ We're good to go!
 $ minikube ip
 192.168.99.100
 
-# Set the `ADDRESS` environemnt variable
-$ export ADDRESS=192.168.99.100:30650
+# Set the `PACHD_ADDRESS` environemnt variable
```

Comment:
Make a note that's something like the below in this section and then we can merge these docs. 

*Note*: ADDRESS was renamed to PACHD_ADDRESS is 1.8.3. If you are using an older version of Pachyderm, use the ADDRESS env var instead.